code-challenges.js

```diff
@@ -6,34 +6,86 @@
 // If you get stuck, please leave comments to help us understand your thought process.
 
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
+var temp = 216
+
+if(temp < 212){
+    console.log(`The temperature is ${temp}, which is below the boiling point`); // logs temp is less than boiling point
+} else if(temp > 212){
+    console.log(`The temperature is ${temp}, which is above the boiling point`); // logs temp is more than boiling point
+} else{
+    console.log(`The temperature is ${temp}, which is at the boiling point`);
+} // logs the temp is equal to the boiling point
+
+// -----NOTE * Since this problem had 3 different outcomes, I used an if else statement. I used the less than greater than operators to allow for any temperature to be given. I tried template literals to help reflect what temp was input by the end user.
+
+
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 42
 // var temp = 350
-// var temp = 212
+// // var temp = 212
+var temp = 42
+if(temp === 42){
+    console.log("42 is below the boiling point");
+} else if(temp === 350){
+    console.log("350 is above the boiling point");
+} else if(temp === 212){
+console.log("212 is at the boiling point");
+}
+//-------NOTE * Piggy backing off the first code, I change the parameters to be more specific to the given variables by using the ===. 
```

Comment:
In this case, we are looking for the first solution. Definitely the right move with the template literal, but keep in mind that if you are given particular inputs and expected outputs, that is what you should produce.